repoauth/pulp/repoauth/protected_repo_utils.py

```diff
@@ -0,0 +1,175 @@
+'''
+This module contains utilities for manipulating a local store of protected
+repo information.
+'''
+
+import os
+from threading import RLock
+
+# -- constants ----------------------------------------------------------------------
+
+WRITE_LOCK = RLock()
```

Comment:
On 02/04/2015 10:06 AM, Michael Hrivnak wrote:
>> +WRITE_LOCK = RLock()
> 
> :scream_cat:

Hahahaha, what the heck is going on in here‽